common/src/main/java/com/wynntils/models/containers/type/ContainerQuickJumpModel.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.containers.type;
```

Comment:
```suggestion
package com.wynntils.models.containers;
```

Move this to the suggested package.

Reply:
Oops